### PR TITLE
Remove triple backticks from RST

### PR DESCRIPTION
**Pull request by jayvdb**
Triple backticks are not RST syntax, resulting in literal backticks
in the output generated by Sphinx.

Fixes https://github.com/coala/coala/issues/4012

**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [ ] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [ ] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `cobot mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

**Comment by Udayan12167** [comment]
ack f0584ae

**Comment by Udayan12167** [comment]
@rultor merge

File: coalib/bearlib/languages/documentation/DocumentationExtraction.py [hunk 1]
```diff
@@ -54,13 +54,11 @@ def _extract_doc_comment_continuous(content, line, column, markers):
 
     The property of the continuous layout is that the each-line-marker and the
     end-marker do equal. Documentation is extracted until no further marker is
-    found. Applies e.g. for doxygen style python documentation:
+    found. Applies e.g. for doxygen style python documentation::
 
-    ```
     ## main
     #
     #  detailed
```
Comment (userzimmermann) [inline coalib/bearlib/languages/documentation/DocumentationExtraction.py]: The code block needs to be indented.

File: coalib/bearlib/languages/documentation/DocumentationExtraction.py [hunk 2]
```diff
@@ -100,13 +98,11 @@ def _extract_doc_comment_standard(content, line, column, markers):
     Extract a documentation that starts at given beginning with standard
     layout.
 
-    The standard layout applies e.g. for C doxygen-style documentation:
+    The standard layout applies e.g. for C doxygen-style documentation::
 
-    ```
     /**
      * documentation
      */
```
Comment (userzimmermann) [inline coalib/bearlib/languages/documentation/DocumentationExtraction.py]: This also needs to be indented.

Comment (Khushmeet) [inline coalib/bearlib/languages/documentation/DocumentationExtraction.py]: mixup of tabs and spaces is there , i think.

**Comment by arjunsinghy96** [comment]
Hey @jayvdb , won't we ideally squash these two commits. Everything else works.

File: coalib/bearlib/languages/documentation/DocumentationExtraction.py [hunk 3]
```diff
@@ -54,13 +54,11 @@ def _extract_doc_comment_continuous(content, line, column, markers):
 
     The property of the continuous layout is that the each-line-marker and the
     end-marker do equal. Documentation is extracted until no further marker is
-    found. Applies e.g. for doxygen style python documentation:
+    found. Applies e.g. for doxygen style python documentation::
```
Comment (satwikkansal) [inline coalib/bearlib/languages/documentation/DocumentationExtraction.py]: unrelated change?

Comment (jayvdb) [inline coalib/bearlib/languages/documentation/DocumentationExtraction.py]: necessary syntax change.

Comment (satwikkansal) [inline coalib/bearlib/languages/documentation/DocumentationExtraction.py]: Don't know if it's a good suggestion or not, but maybe some description of this change should be included in commit message description (currently it mentions only triple backticks) :sweat_smile:

Comment (jayvdb) [inline coalib/bearlib/languages/documentation/DocumentationExtraction.py]: The first line of this commit message is "Use correct RST syntax".  Currently it is using markdown syntax.

Comment (jayvdb) [inline coalib/bearlib/languages/documentation/DocumentationExtraction.py]: ok clarified this..i hope

Comment (satwikkansal) [inline coalib/bearlib/languages/documentation/DocumentationExtraction.py]: Yes.. Thanks :)

Comment (jayvdb) [inline coalib/bearlib/languages/documentation/DocumentationExtraction.py]: Need an ack to proceed..

**Comment by satwikkansal** [comment]
ack 66fabb6

**Comment by userzimmermann** [comment]
ack https://github.com/coala/coala/pull/4013/commits/07298a536e18981f8ac033142cb5b6f6d5788e2d

**Comment by userzimmermann** [comment]
@rultor merge

**Comment by SanketDG** [comment]
needs rebase

**Comment by jayvdb** [comment]
reack e95eefa 5447f50

**Comment by jayvdb** [comment]
@rultor merge
